 Albeit both FT_Get_{First,Next}_Char seem to exist in freetype
 library, calling them, at least from mplayer, results in linkage
 undefined references. FT_Get_Char_Index links fine, but the cost
 is a slower MPlayer (with subtitles) startup. Any bright ideas?

--- ./libvo/font_load_ft.c.orig	2003-01-06 14:23:59.000000000 +0200
+++ ./libvo/font_load_ft.c	2003-01-26 14:41:20.000000000 +0200
@@ -30,7 +30,7 @@
 #include "osd_font.h"
 
 #if (FREETYPE_MAJOR > 2) || (FREETYPE_MAJOR == 2 && FREETYPE_MINOR >= 1)
-#define HAVE_FREETYPE21
+// #define HAVE_FREETYPE21
 #endif
 
 char *get_path(char *filename);
